model/security/get_distinguished_name/structures.smithy

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+
+@input
+structure GetDistinguishedName_Input{
+    @required
+    @httpLabel
+    cluster_name: String
+}
+
+@output
+structure GetDistinguishedName_Output {
+    distinguishedname: DistinguishedName
```

Comment:
This returns a map:
```suggestion
    @httpPayload
    content: DistinguishedNamesMap
```

It is expected this will cause build errors until #127 is merged